src/studio/src/designer/frontend/packages/schema-editor/src/components/StyledSelect.tsx

```diff
@@ -30,19 +28,19 @@ const useStyles = makeStyles({
   },
 });
 
-export const TypeSelect = (props: ITypeSelectProps) => {
+export const StyledSelect = (props: ITypeSelectProps) => {
   const classes = useStyles();
   const {
     id, value, onChange,
   } = props;
 
   const onValueChange = (event: any) => {
-    onChange(id, event.target.value);
+    onChange(event.target.value);
   };
 
   return (
     <Select
-      id={`${getDomFriendlyID(id)}-type-select`}
+      id={id}
```

Comment:
This should solve some bug I guess? :)

Reply:
No, I just found it odd that the component itself had to make the id dom friendly. Should be valid when passed imo.